x-pack/plugins/security_solution/common/search_strategy/timeline/events/last_event_time/index.ts

```diff
@@ -25,6 +25,15 @@ export interface TimelineEventsLastEventTimeStrategyResponse extends IEsSearchRe
   inspect?: Maybe<Inspect>;
 }
 
+export interface TimelineKpiStrategyResponse extends IEsSearchResponse {
+  destinationIpCount: Maybe<number>;
+  inspect?: Maybe<Inspect>;
+  hostCount: Maybe<number>;
```

Comment:
All of these should technically be optional since `Maybe<T> = T | null`

Reply:
Yes I agree, you do not need the Maybe or the optional since you will have 0 or the value.

For history, the Maybe is coming from the graphql when we had both type at the same time